src/libtego_ui/protocol/Connection.cpp

```diff
@@ -465,7 +467,7 @@ int ConnectionPrivate::availableOutboundChannelId()
 
     // Find an unused id, trying a maximum of 100 times, using a random step to avoid collision
     for (int i = 0; i < 100 && channels.contains(nextOutboundChannelId); i++) {
-        nextOutboundChannelId += 1 + (qrand() % 200);
+        nextOutboundChannelId += 1 + ((int)QRandomGenerator::global()->generate64() % 200);
```

Comment:
If we're looking for an int on [0,199] than we should use QRandomGenerator::bounded(0, 199) ( see: https://doc.qt.io/qt-5/qrandomgenerator.html#bounded-4 )

Reply:
QRandomGenerator::global()->bounded(200) works as well. `highest` argument to bounded is exclusive, so this will generate from 0-199 inclusive.